libraries/AP_Mission/AP_Mission.cpp

```diff
@@ -1815,11 +1831,11 @@ bool AP_Mission::mission_cmd_to_mavlink_int(const AP_Mission::Mission_Command& c
         packet.param1 = cmd.p1;
         break;
 
-    case MAV_CMD_DO_GIMBAL_MANAGER_PITCHYAW:
-        packet.param1 = cmd.content.gimbal_manager_pitchyaw.pitch_angle_deg;
-        packet.param2 = cmd.content.gimbal_manager_pitchyaw.yaw_angle_deg;
-        packet.param3 = cmd.content.gimbal_manager_pitchyaw.pitch_rate_degs;
-        packet.param4 = cmd.content.gimbal_manager_pitchyaw.yaw_rate_degs;
+    case MAV_CMD_DO_GIMBAL_MANAGER_PITCHYAW: 
+        packet.param1 = (fabs(packet.param1 - INT8_MAX) < FLT_EPSILON) ? NAN : cmd.content.gimbal_manager_pitchyaw.pitch_angle_deg;
```

Comment:
AFAICS packet.param1 will always be zero here, so checking it for any other value makes no sense.

Reply:
I corrected the issue in my newest commit